.github/workflows/unit-tests.yml

```diff
@@ -36,6 +36,3 @@ jobs:
         run: npm run test:ci
         env:
           CI: true
-
-      - name: Upload result to codecov
-        run: npm run codecov
```

Comment:
I think nobody's been looking at codecov stats for months now so if there's no veto, I would just remove it. Also removing the `codecov.yml` file at the repo root.

Reply:
No veto here! 🙂👍